pkg/controller/launcher/launcher_controller.go

```diff
@@ -172,12 +172,12 @@ func (r *ReconcileLauncher) Reconcile(request reconcile.Request) (reconcile.Resu
 		}
 
 		for _, gitConfig := range instance.Spec.Git.GitProviders {
-		index := findByID(gitProviders, gitConfig.ID)
+			index := findByID(gitProviders, gitConfig.ID)
 			if index == -1 {
 				return reconcile.Result{}, fmt.Errorf("could not find git provider config with ID: '%s'", gitConfig.ID)
 			}
-		gitProviders[index].ClientProperties.ClientID = gitConfig.ClientID
-		gitProviders[index].ServerProperties.ClientSecret = gitConfig.ClientSecret
+			gitProviders[index].ClientProperties.ClientID = gitConfig.ClientID
+			gitProviders[index].ServerProperties.ClientSecret = gitConfig.ClientSecret
```

Comment:
Indentation changes are unintended. I guess my IDE auto-corrected these lines to match the rest of the file.
Let me know if you want that reverted.

Reply:
@matskiv That is something that go does automatically and it's build into the language, so it's fine.